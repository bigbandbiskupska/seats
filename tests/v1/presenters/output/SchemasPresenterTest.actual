[
	1 => [
		'id' => 1,
		'name' => 'Vánoční koncert',
		'max_allowed' => 5,
		'price' => 250,
		'hidden' => 1,
		'locked' => 0,
	],
	[
		'id' => 2,
		'name' => 'Velikonoční koncert',
		'max_allowed' => 5,
		'price' => 100,
		'hidden' => 0,
		'locked' => 1,
	],
]
