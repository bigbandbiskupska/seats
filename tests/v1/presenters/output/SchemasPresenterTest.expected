[
	1 => [
		'id' => 1,
		'name' => 'Vánoční koncert',
		'price' => 250,
		'hidden' => 1,
		'locked' => 0,
	],
	[
		'id' => 2,
		'name' => 'Velikonoční koncert',
		'price' => 100,
		'hidden' => 0,
		'locked' => 1,
	],
]
